app/helpers/era_av_helper.rb

```diff
@@ -0,0 +1,29 @@
+# Copyright 2017, University of Alberta Libraries.
```

Comment:
That's valid then. Seems silly but if that's what the community is doing then guess you want to follow their standards so that's fair enough.